ch01.asciidoc

```diff
@@ -86,6 +86,11 @@ Mining for bitcoin::
 
 Each of these stories is based on the real people and real industries currently using bitcoin to create new markets, new industries, and innovative solutions to global economic issues.((("", startref="GSuses01")))
 
+[TIP]
```

Comment:
@krupawan5618 

This is great! Let's change:

> Have you recently checked the bitcoin exchange rate for your desired currency pair? As explained in <<bitcoin_price>>, the price of bitcoin is moving! As explained in <<bitcoin_price>>, the price of bitcoin is moving!

To:

> The price of bitcoin has changed a lot over time. As of March 2021, a person would need approximately $54000 USD to purchase one whole bitcoin.

Reply:
👍